### DISCUSSION TITLE
Can I make a PR for supporting django4.2 stable version

### DISCUSSION THREAD

**Discussion by simkimsia** [torchbox/django-pattern-library#228] [General]
I did this PR 5 months ago which was just merged

https://github.com/torchbox/django-pattern-library/pull/220

Since then 4.2 has gone stable

If it's all right, I like to make a PR for upgrading the support for 4.2 from 4.2rc1 to actually 4.2

@thibaudcolas ok?

**Comment by thibaudcolas** [comment]
Please just go ahead with PRs in a scenario like this (or open an issue) :)

**Reply by zerolab** [reply]
Note that full 4.2 support was formalized in #231
